pkg/planner/core/plan_cache_utils.go

```diff
@@ -502,6 +503,29 @@ func GetPreparedStmt(stmt *ast.ExecuteStmt, vars *variable.SessionVars) (*PlanCa
 	return nil, plannererrors.ErrStmtNotFound
 }
 
+func init() {
+	debugtrace.GetPreparedStmt = GetPreparedStmt4DebugTrace
```

Comment:
this part is about avoiding debugTrace pkg depending on `core.PlanCacheStmt`. Only core pkg use debugTrace pkg is reasonable.